crypto/ake/enclave/src/lib.rs

```diff
@@ -166,9 +166,10 @@ impl<EI: EnclaveIdentity> AkeEnclaveState<EI> {
         let mut csprng = McRng::default();
         // TODO: Cache expected values rather than creating a new report each time.
         let report_body = Report::new(None, None)?.body();
+        let expected_measurements: [Measurement; 1] = [report_body.mr_enclave().into()];
```

Comment:
Is this the preferred way to create a `Vec` in the enclave? (just curious)

Reply:
Actually, I'm reading @jcape's [comments](https://github.com/mobilecoinofficial/mobilecoin/pull/188#discussion_r432186289) above. I think my confusion at the moment is that it goes `fixed length array` (via initializer literal) -> `Vec` (via `to_vec`) -> `&[Measurement]` (via `&`).

My initial comment was about why create a fixed length array first before creating a `Vec`, but after reading @jcape's [comment](https://github.com/mobilecoinofficial/mobilecoin/pull/188#discussion_r432186289), I think my issue is more around having the `to_vec`, and I believe his comment is suggesting not using `Vec` at all in this function.

Reply:
Thank you as well for this explanation - made the fixes!